src/js/gi-bill-comparison-tool/components/Dropdown.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+class Dropdown extends React.Component {
```

Comment:
There is `<ErrorableSelect/>` but I didn't want to bite off trying to reuse that before I knew what that css style implications of that would be. Certainly open to reusing/extending that if people agree it makes sense (as a future enhancement.)